### PR TITLE
Add new opcodes: renameat, unlinkat, mkdirat, symlinkat, linkat

### PR DESCRIPTION
**Pull request by spikat**
Here is the addition of the latest opcodes, plus an example for each of them. :)
Validated on my side on a Ubuntu 22.04, running 5.15.0-27-generic kernel.

### CODE REVIEW

**Comment by Iceber** [comment]
Thank you very much for adding these latest features and examples, I will review them as soon as possible.

File: prep_request.go [hunk 1]
```diff
@@ -650,3 +651,131 @@ func EpollCtl(epfd int, op int, fd int, event *syscall.EpollEvent) PrepRequest {
...
+	bLink, err := syscall.ByteSliceFromString(linkPath)
+	if err != nil {
+		return nil, err
+	}
+
+	bpTarget := unsafe.Pointer(&bTarget[0])
+	bpLink := unsafe.Pointer(&bLink[0])
+	return func(sqe *iouring_syscall.SubmissionQueueEntry, userData *UserData) {
+		userData.hold(&bTarget)
+		userData.Hold(&bLink)
```
Comment (Iceber) [inline prep_request.go:712]: The `userData.hold` could set multiple parameters
```go
userData.hold(&bTarget, &bLink)
```

Comment (spikat) [reply prep_request.go]: Fixed on last commit :)

**Review by Iceber — APPROVED** [review]
LGTM, Thank you very much for contributing! :heart:

**Comment by spikat** [comment]
You're welcome! Thanks for your quick review/merge 😃

**Pull request merged**